pandapipes/pf/pipeflow_setup.py

```diff
@@ -619,39 +637,3 @@ def reduce_pit(net, node_pit, branch_pit, nodes_connected, branches_connected):
             from_to_active_lookup[table] = (count, count + len_new)
             count += len_new
         net["_lookups"]["%s_from_to_active" % el] = from_to_active_lookup
-
-
-def extract_results_active_pit(net, node_pit, branch_pit, nodes_connected, branches_connected):
-    """
-    Extract the pipeflow results from the internal pit structure ("_active_pit") to the general pit
-    structure.
-
-    :param net: The pandapipes net that the internal structure belongs to
-    :type net: pandapipesNet
-    :param node_pit: The internal structure node array
-    :type node_pit: np.array
-    :param branch_pit: The internal structure branch array
-    :type branch_pit: np.array
-    :param nodes_connected: A mask array stating which nodes are actually connected to the rest of\
-            the net
-    :type nodes_connected: np.array
-    :param branches_connected: A mask array stating which branches are actually connected to the \
-             rest of the net
-    :type branches_connected: np.array
-    :return: No output
-
-    """
-    if not np.alltrue(nodes_connected):
-        node_pit[~nodes_connected, PINIT] = np.NaN
-        node_pit[nodes_connected, :] = net["_active_pit"]["node"]
-    else:
-        net["_pit"]["node"] = np.copy(net["_active_pit"]["node"])
-    if not np.alltrue(branches_connected):
-        branch_pit[~branches_connected, VINIT] = np.NaN
-        rows_active_br = np.where(branches_connected)[0]
-        cols_br = np.array([i for i in range(branch_pit.shape[1])
-                            if i not in [FROM_NODE, TO_NODE, FROM_NODE_T, TO_NODE_T]])
-        branch_pit[rows_active_br[:, np.newaxis], cols_br[np.newaxis, :]] = \
-            net["_active_pit"]["branch"][:, cols_br]
-    else:
-        net["_pit"]["branch"] = np.copy(net["_active_pit"]["branch"])
```

Comment:
This is moved to pf/result_extraction. I think, it makes more sense to define this function there. Maybe it would even make sense to split the pipeflow_setup module into other modules with more precise names.